index.html

```diff
@@ -0,0 +1,134 @@
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+            <li class="container2">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot2.png" alt=""></li>
+                    <li class="works-heading">Multi-Post Stories</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+            <li class="container1">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot3.png" alt=""></li>
+                    <li class="works-heading">Tonic</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+            <li class="container2">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot4.png" alt=""></li>
+                    <li class="works-heading">Multi-Post Stories</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+        </ul>
+    </section>
```

Comment:
- You have done well so far, but please note that it is essential to use the appropriate semantic HTML tags when necessary. The `ul` tag should be used when there is a need to list items. Otherwise, I suggest you consider using the `div` tag or `article` tag when there is a need to group items. Please consider this and fix it for all project cards 🙏 🙏. According to the project requirements, the container for all your project cards should be a `section` tag
 - You can consider using either a `div` or `article` tag as the container for each of your project cards, this would help improve the semantics of your page 😄